MAUI/Circular-Charts/DoughnutChart.md

```diff
@@ -103,4 +103,58 @@ chart.Series.Add(series);
 
 {% endtabs %}
 
-![Semi doughnut chart in MAUI Chart](Chart-Types_images/maui_semi_doughnut_chart.png)
\ No newline at end of file
+![Semi doughnut chart in MAUI Chart](Chart-Types_images/maui_semi_doughnut_chart.png)
+
+## Center View
+
+The view placed in the center of the doughnut chart is useful for sharing additional information about the doughnut chart. Any view can be added to the center of the doughnut chart using the [CenterView]() property of [DoughnutSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DoughnutSeries.html). The binding context of the [CenterView]() will be the respective DoughnutSeries.
+
+### Center Hole Size
+
+The [CenterHoleSize]() property of [DoughnutSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DoughnutSeries.html) is used to get the diameter value of the doughnut center hole. Using this [CenterHoleSize]() value, you can provide [CentreView]() for the series to prevent the view from overlapping the series.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCircularChart>    
+    <chart:DoughnutSeries ItemsSource="{Binding Data}" XBindingPath="Name" YBindingPath="Value"/>
+        <chart:DoughnutSeries.CenterView>
+                <Border HeightRequest="{Binding CenterHoleSize}" WidthRequest="{Binding CenterHoleSize}">
+                    <Border.StrokeShape>
+                        <RoundRectangle CornerRadius="{Binding CenterHoleSize,Converter={StaticResource innerRadiusConverter}"/>
+                    </Border.StrokeShape>
```

Comment:
mentioned converter in code, but no code regarding this. 

For UG we can set static value for the corner radius. 
`<Border.StrokeShape>
                        <RoundRectangle CornerRadius="200"/>
                    </Border.StrokeShape>`

Reply:
Complete the changes and update PR